python/python/glide/async_commands/cluster_commands.py

```diff
@@ -920,6 +920,64 @@ async def publish(
         )
         return cast(int, result)
 
+    async def pubsub_shardchannels(
+        self, pattern: Optional[TEncodable] = None
+    ) -> List[bytes]:
+        """
+        Lists the currently active shard channels.
+
+        See https://valkey.io/commands/pubsub-shardchannels for details.
+
+        Args:
+            pattern (Optional[TEncodable]): A glob-style pattern to match active shard channels.
+                                If not provided, all active shard channels are returned.
+
+        Returns:
+            List[bytes]: A list of currently active shard channels matching the given pattern.
```

Comment:
we have to describe the routing and aggregation logic here - how is the client supposed to know how this command is routed?